index.html

```diff
@@ -70,8 +70,8 @@
 of each technical contribution to the web
 in a distributed and modular way. 
 
-Version numbers such as "Web 2.0" have been used
-for educational, communications or marketing purposes,
+Version numbers such as "Web 2.0", "Web 3.0", "Web3", or "Web5" have been used
+for educational, communications, or marketing purposes,
 which is different from versioning in the sense that software is versioned.
```

Comment:
These additional examples don't fit in this sentence. They belong to the next sentence.

Reply:
They've all been used " for educational, communications, or marketing purposes" (and that's being kind). They fit there. None of them uniquely deserve to be separated out.